### PR TITLE
XP-228 Update readme

### PR DESCRIPTION
**Pull request by wilk10**
### References

[XP-228](https://xainag.atlassian.net/browse/XP-228)

### Summary

Updated the README to be consistent with what this repo will become, ahead of the removal of benchmarking from here.

### Are there any open tasks/blockers for the ticket?

None

---

### Reviewer checklist

*Reviewer agreement:*

* Reviewers assign themselves at the start of the review.
* Reviewers do **not** commit or merge the merge request.
* Reviewers have to check and mark items in the checklist.

**Merge request checklist**

- [ ] Conforms to the merge request title naming `XP-XXX <a description in imperative form>`.
- [ ] Each commit conforms to the naming convention `XP-XXX <a description in imperative form>`.
- [ ] Linked the ticket in the merge request title or the references section.
- [ ] Added an informative merge request summary.

**Code checklist**

- [ ] Conforms to the branch naming `XP-XXX-<a_small_stub>`.
- [ ] Passed scope checks.
- [ ] Added or updated tests if needed.
- [ ] Added or updated code documentation if needed.
- [ ] Conforms to Google docstring style.
- [ ] Conforms to XAIN structlog style.

### CODE REVIEW

**Comment by wilk10** [comment]
The ticket asks to specify how to setup a virtualenv, but i guess we can take for granted that they know how to do it, given the audience we have?

**Comment by wilk10** [comment]
added the coordinator CLI info, as @Robert-Steiner asked, good idea

File: README.md [hunk 1]
```diff
@@ -77,6 +55,20 @@ The generated documentation will be under `docs/_build/html/`. You can open the
 root of the documentation by opening `docs/_build/html/index.html` on your
 favorite browser.
 
+### Running the Coordinator locally
+
+To run the Coordinator on your local machine, use the command:
+
+```shell
+$ python xain_fl/grpc/cli.py -f test_array.npy
```
Comment (Robert-Steiner) [inline README.md]: Would be nice to know how to generate the .npy file from a ml model (at least for keras).

Comment (wilk10) [reply README.md]: i think it's a good idea but is it ok if we create a separate ticket for this because it's outside of the scope of this task?

Comment (Robert-Steiner) [reply README.md]: That's is a good idea

Comment (wilk10) [reply README.md]: https://xainag.atlassian.net/browse/XP-298. thanks for approving!

**Pull request merged**